agent/router/router.go

```diff
@@ -341,6 +357,16 @@ func (r *Router) FindRoute(datacenter string) (*Manager, *metadata.Server, bool)
 	return r.routeFn(datacenter)
 }
 
+func (r *Router) FindLANRoute() (*Manager, *metadata.Server, bool) {
```

Comment:
Outside of tests all callers of this function ignore the bool return value. I think we could remove it, since callers are able to check if the manager or server are nil (which is what they do now).

Reply:
Removed the bool.